tests/main/devtools-spec.ts

```diff
@@ -1,34 +1,54 @@
 import { setupDevTools } from '../../src/main/devtools';
 
-let mockIsDevMode = false;
-
-jest.mock('../../src/utils/devmode', () => ({
-  get isDevMode() {
-    return mockIsDevMode;
-  }
-}));
-
 jest.mock('electron-devtools-installer', () => ({
   default: jest.fn(),
   REACT_DEVELOPER_TOOLS: 'REACT_DEVELOPER_TOOLS',
   REACT_PERF: 'REACT_PERF'
 }));
 
 describe('devtools', () => {
+  const old = (process as any).defaultApp; // for tsconfig error
+
+  afterEach(() => {
+    Object.defineProperty(process, 'defaultApp', { value: old });
+  });
+
   it('does not set up developer tools if not in dev mode', () => {
     const devtools = require('electron-devtools-installer');
-
+    Object.defineProperty(process, 'defaultApp', {
```

Comment:
Instead of setting `process.defaultApp` and calling the actual `isDevMode()` function in the tests, could you use the `jest.Mock()` approach to instead mock the return value of the `isDevMode()` function.

The motivation is that we don't want to test if `isDevMode()` returns the correct value of `process.defaultApp`. Rather, assuming that `isDevMode()` returns a certain results, we want to test how our function works.

See code below:
```ts
import { isDevMode } from '../../src/utils/devmode';
jest.mock('../../src/utils/devmode');
// ...
it('sets up developer tools if in dev mode', () => {
    // ...
    (isDevMode as jest.Mock).mockReturnValue(true); // mock false value for the other test
});
```